### ISSUE TITLE
certificate link in prov_dev_client_Sample

### ISSUE THREAD

**Issue by omersalam** [Azure/azure-iot-sdk-c#1517]
Hello 
https://docs.microsoft.com/en-us/azure/iot-dps/quick-create-simulated-device-x509

The documentation does not explain how to use a custom cert to attest onto the DPS from C.

Using the example cert works fine. You run dice_device_enrollment, register the generated cert as an individual enrollment and then run prov_dev_client_sample and everything is great.

However it's not at all clear how to use a cert which has already been generated (and so use a group enrollment). With the IoT client sample you can use IoTHubDeviceClient_LL_SetOption with OPTION_X509_CERT and OPTION_X509_PRIVATE_KEY. With the DPS client sample in this file I can find nothing.

To summarise: how does the code in prov_dev_client_sample access the certificate generated in dice_device_enrollment and how can I give it my X509 certificate instead?

**Comment by wesmc7777** [comment]
Please see [https://github.com/MicrosoftDocs/azure-docs/issues/27534](https://github.com/MicrosoftDocs/azure-docs/issues/27534). I'll try to keep all comments in a single location

**Comment by ewertons** [comment]
Hi @omersalam ,

We apologize for the long delay, but we were assigned to work on new code development and that affected our github response time.

Did you have chance to get your question addressed by the link posted by @wesmc7777 ?

We will close this issue for now, but if you would like it to be reopened and follow up on this matter, please post a comment requesting so.

Thanks,
Azure IoT SDK Team.

**Comment by az-iot-builder-01** [comment]
@omersalam, thank you for your contribution to our open-sourced project! Please help us improve by filling out this 2-minute [customer satisfaction survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbRx8jpG79OudPqY-BojcKjkJUMFFBQjgxNkw5ODJIRjBWTzI0Nk85R0pUQiQlQCN0PWcu)
